scripts/release-post.js

```diff
@@ -24,48 +24,33 @@ const fs = require('fs')
 const path = require('path')
 const Handlebars = require('handlebars')
 const request = require('request')
-const changelogUrl = require('changelog-url')
-const semver = require('semver')
 
 const downloads = require('./helpers/downloads')
 
-function sendRequest (uri, method) {
+function sendRequest (options) {
   return new Promise((resolve, reject) => {
-    request({
-      headers: { 'User-Agent': 'nodejs.org release blog post script' },
-      method: method,
-      uri: uri
-    }, (err, res, body) => {
+    request(Object.assign({
+      headers: { 'User-Agent': 'nodejs.org release blog post script' }
+    }, options), (err, res, body) => {
```

Comment:
Could the result `Object.assign()` be assigned to a variable before executing `request()` here?